### PR TITLE
Added test_default_ping_timer_and_epoll_thread_count

### PR DESCRIPTION
**Pull request by aujjwal-redhat**
Migrated the [test case](https://github.com/gluster/glusto-tests/blob/HEAD/tests/functional/glusterd/test_default_ping_timer_and_epoll_thread_count.py)
Added the redant functionalities wherever needed.

Updates : #292

Fixes : #309

Signed-off-by: Ayush Ujjwal <aujjwal@redhat.com>

### CODE REVIEW

**Comment by aujjwal-redhat** [comment]
Screenshot from 2021-05-11 21-14-34

File: tests/functional/glusterd/test_default_ping_timer_and_epoll_thread_count.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        Test Steps:
+        1. Start glusterd
+        2. Check ping timeout value in glusterd.vol should be 0
+        3. Create a test script for epoll thread count
+        4. Source the test script
+        5. Fetch the pid of glusterd
+        6. Check epoll thread count of glusterd should be 1
+        """
+        # Fetch the ping timeout value from glusterd.vol file
+        cmd = "cat /usr/local/etc/glusterfs/glusterd.vol |\
```
Comment (nik-redhat) [inline tests/functional/glusterd/test_default_ping_timer_and_epoll_thread_count.py:41]: Are we using source installation or RPM installation in nightly test machines?

Comment (aujjwal-redhat) [reply tests/functional/glusterd/test_default_ping_timer_and_epoll_thread_count.py:41]: @srijan-sivakumar  ?

Comment (nik-redhat) [reply tests/functional/glusterd/test_default_ping_timer_and_epoll_thread_count.py:41]: @srijan-sivakumar are we planning to move to RPm installation anytime soon or there is some time due for that? If so, we can add a TODO and move ahead with this PR.

Comment (srijan-sivakumar) [reply tests/functional/glusterd/test_default_ping_timer_and_epoll_thread_count.py:41]: We should be moving to rpm install. I think this is required in the coming days. Let's take this part of iteration 4. @aujjwal-redhat take a specific OS and work on it for now. Say fedora 32. We can check other distros later

Comment (aujjwal-redhat) [reply tests/functional/glusterd/test_default_ping_timer_and_epoll_thread_count.py:41]: Ack

**Review by srijan-sivakumar — APPROVED** [review]
LGTM

**Comment by nik-redhat** [comment]
Are....todo toh daal dete 😂

**Comment by aujjwal-redhat** [comment]
Sirf todo nahi changes karne h XD

**Comment by aujjwal-redhat** [comment]
Adding a patch XD

**Comment by srijan-sivakumar** [comment]
Draft.. :p Next time..lets do that..for now..added this as an exclusion. We'll fix it.

**Pull request merged**